CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,9 @@
 == 3030-12-31 Getting Started
 <p>If you're new to CONSTELLATION, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.introduction">introduction</a>.</p>
 
+== 2021-05-19 Added the ability to pin the position of vertexes to prevent them from moving when using arrangement plugins 
+<p>Vertexes can now be pinned, either through the pin/unpin toolbar icons in the arrangements panel, or via the corresponding Arrange menu options. Once pinned Vertexes do not move when arrangement plugins are run, but can be manually moved. The 'pinned' vertex stores this information.</p>
```

Comment:
Change vertexes to nodes. I also assume the last sentence is meant to say 'pinned' attribute